pages/phases/index.html

```diff
@@ -121,16 +121,16 @@ <h2 class="chp-card__header">
   <div class="grid-container">
     <div class="grid-row grid-gap">
       <div class="desktop:grid-col-6">
-        <a href="{{ site.baseurl }}/hiring-phases/phone-assessment-interviews/" class="chp-card">
+        <a href="{{ site.baseurl }}/hiring-phases/assessments/" class="chp-card">
           <div class="chp-card__figure">
-            <img src="{{ site.baseurl }}/assets/img/icon-phone-interviews.png" alt="Depiction of job applicants" />
+            <img src="{{ site.baseurl }}/assets/img/icon-assessments.png" alt="Depiction of job applicants" />
           </div>
           <div class="chp-card__body">
             <h2 class="chp-card__header">
-              Phone Interviews
+              Assessments
             </h2>
             <p>
-              SMEs conduct up to two rounds of phone interviews to determine which applicants meet the minimum qualifications.
+              SMEs conduct up to two rounds of assessments to determine which applicants meet the minimum qualifications.
```

Comment:
should we say "verify" vs "determine"?

Reply:
Done: https://github.com/usds/SME-QA-Website/pull/243/commits/635a560848b8830bd12f88d03028fb9cb47ace2b